src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseImportService.java

```diff
@@ -199,7 +199,7 @@ private void cloneAndEnableAllBuildPlans(ProgrammingExercise templateExercise, P
         final var templateKey = templateExercise.getProjectKey();
         final var targetKey = newExercise.getProjectKey();
         final var targetName = newExercise.getCourseViaExerciseGroupOrCourseMember().getShortName().toUpperCase() + " " + newExercise.getTitle();
-        continuousIntegrationService.get().createProjectForExercise(newExercise);
+        continuousIntegrationService.orElseThrow().createProjectForExercise(newExercise);
         continuousIntegrationService.get().copyBuildPlan(templateKey, templatePlanName, targetKey, targetName, templatePlanName, false);
         continuousIntegrationService.get().copyBuildPlan(templateKey, solutionPlanName, targetKey, targetName, solutionPlanName, true);
         continuousIntegrationService.get().givePlanPermissions(newExercise, templatePlanName);
```

Comment:
This is one example of bad code style as described in the PR description. To keep this PR simple, I'll refactor this in a Followup-PR.